src/config/database.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+$host = "localhost"; // Your MySQL host
+$username = "root"; // Your MySQL username
```

Comment:
Let's note that in the future we should be providing this values through environment variables or similar